geopandas/io/sql.py

```diff
@@ -248,7 +248,11 @@ def _convert_to_ewkb(gdf, geom_name, srid):
 
         geoms = [dumps(geom, srid=srid, hex=True) for geom in gdf[geom_name]]
 
-    gdf[geom_name] = geoms
+    # The gdf will warn that the geometry column doesn't hold in-memory geometries
+    # now that they are EWKB. Ignore this warning.
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=UserWarning)
```

Comment:
```suggestion
        warnings.filterwarnings("ignore", "Geometry column does not contain geometry.", UserWarning)
```

Can we catch that specific warning only?

Reply:
Yep, makes sense

Reply:
Ah, sorry. I made the line too long, `black` and `flake8` [don't like it](https://travis-ci.org/github/geopandas/geopandas/jobs/703264384#L3772). Can you [blacken](https://geopandas.readthedocs.io/en/latest/contributing.html#style-guide-linting) the code?